templates/base/package.json

```diff
@@ -2,7 +2,8 @@
   "name": "base",
   "version": "0.1.0",
   "private": true,
-  "scripts": {
+    "scripts": {
```

Comment:
We don't need these changes here. See if you need to bring any improvements here quality Control Repo. These configurations will be added via that package. https://github.com/pagesource/quality-control